pkg/cmd/tknpac/bootstrap/bootstrap.go

```diff
@@ -222,30 +227,31 @@ func GithubApp(run *params.Run, ioStreams *cli.IOStreams) *cobra.Command {
 }
 
 func DetectPacInstallation(ctx context.Context, wantedNS string, run *params.Run) (string, error) {
-	// detect which namespace pac is installed in
-	// verify first if the targetNamespace actually exists
-	if wantedNS != "" {
-		installed, err := checkNS(ctx, run, wantedNS)
-		if err != nil {
-			return "", err
-		}
-		if !installed {
-			return "", fmt.Errorf("PAC is not installed in namespace: %s", wantedNS)
-		}
-		return wantedNS, nil
-		// if openshift pipelines ns is installed try it from there
+	_, err := run.Clients.Extension.ApiextensionsV1().
```

Comment:
was wondering if we should just keep configmap check below to detect pac installation 🤔 
it will make sure pac is installed as well as detect the namespace
wdyt?

Reply:
if hopefully someone is not calling the same configmap name :-D  (but we can make sure it is by checking the annotations too)